docs/src/examples/components/Text/Types/TextSizesExample.tsx

```diff
@@ -7,7 +7,7 @@ const TextSizesExample = () => (
     render={({ siteVariables }) => {
       return _.map(siteVariables.fontSizes, (value, key) => (
         <div key={key}>
-          <Text size={key}>This is size="{key}" size font.</Text>
+          <Text size={key as any}>This is size="{key}" size font.</Text>
```

Comment:
creating task for having a unified Interface for both the `siteVariables.fontSizes` and their use in the components, which is now `'small' | 'big' ...`